src/components/AuthorCardEditing.vue

```diff
@@ -14,7 +14,8 @@
                 standout
                 title="The person's given names."
                 v-bind:model-value="givenNames"
-                v-bind:rules="[validateGivenNames]"
+                v-bind:error="fieldErrors.givenNames.length > 0"
```

Comment:
It seems an empty string is used to signify there are no errors.
As an empty string is also falsy, can we simplify to
```suggestion
                v-bind:error="fieldErrors.givenNames"
```